src/cartesian/CartesianAxis.tsx

```diff
@@ -43,6 +43,8 @@ export interface CartesianAxisProps {
   tickFormatter?: (value: any, index: number) => string;
   ticksGenerator?: (props?: CartesianAxisProps) => CartesianTickItem[];
   interval?: number | 'preserveStart' | 'preserveEnd' | 'preserveStartEnd' | 'equidistantPreserveStart';
+  angle?: number;
```

Comment:
Why add the angle here? Seems like an unrelated change.

Reply:
I was going off of the property being present in the documentation (https://recharts.org/en-US/api/XAxis#angle) but now that I looked at it again this is only the case for the XAxis component.

Also I do wonder whether it should be 🤔 Because it might be a bit misleading to expose properties that are limited to affecting the ticklabels on the main axis components.